db/migrate/20200904005346_create_genres.rb

```diff
@@ -0,0 +1,13 @@
+class CreateGenres < ActiveRecord::Migration[5.2]
+  def change
+    create_table :genres do |t|
+      t.string :name
```

Comment:
Let's add some database protections to ensure we can't put genres with empty names in the database.

To do that, add `null: false` to this line. EG: `t.string :name, null: false`

You can re-run your migrations with either:

1. `rails db:migrate:rollback` and `rails db:migrate`, or
2. `rails db:migrate:redo`